docs/pages/docs/providers/rust.md

```diff
@@ -46,3 +46,12 @@ These directories are cached between builds
 - Build: `~/.cargo/git`
 - Build: `~/.cargo/registry`
 - Build: `target`
+
+## [Workspaces](https://doc.rust-lang.org/book/ch14-03-cargo-workspaces.html)
+
+Nixpacks will auto-detect if you are using Cargo Workspaces.
+This checks `workspace.default_members` first and then `workspace.members`.
+It also respects the `workspace.exclude` field.
+
+To set which workspace Nixpacks will build, just set the `NIXPACKS_CARGO_WORKSPACE`
```

Comment:
I think this is actually specifying the _crate_ within the workspace, not the workspace.  Seems like an awkward name for the environment variable.